scripts/linters/ts_ignore_exceptions.json

```diff
@@ -0,0 +1,145 @@
+{
+  "core/templates/domain/improvements/NeedsGuidingResponsesTaskObjectFactorySpec.ts": [
+    "task_type: '???',"
+  ],
+  "extensions/interactions/LogicProof/static/js/tools/logic-demo-test.controller.spec.ts": [
```

Comment:
Also -- one general note, please try to remove ts-ignores wherever possible (again, per  https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-6.html).

I think that, in the comment that accompanies the ts-ignore, you should make it a convention to clearly include the error message that shows up when the ts-ignore is removed, followed by an explanation. This will help developers understand what exactly the problem is and allow reviewers to evaluate if it is reasonable or not.

Please fix for all instances of ts-ignore that remain in this file. You can use a format like

```
@ts-ignore This throws a "Property 'camelCase' is missing in type '(selector: any, context: any) => any'" error. We need to manually suppress this error because ...
```

and have all your comments be in a similar format, and have the lint check require this format too.

Reply:
done. Also I've used the convention
```
// Explanation why the error is caused. Why we need to suppress it.
// @ts-ignore
```
This way it will be easier for the lint check to check for the explanation.

Reply:
Yes, that convention seems fine, thanks!